### PR TITLE
Doc: guide for API usage of previous OSP commands

### PR DESCRIPTION
**Pull request by jjnicola**
**What**:
Document API usage of previous OSP commands
Jira: SC-1006

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] PR merge commit message adjusted

### CODE REVIEW

File: rust/openvasd/doc/osp-cmd-equivalence.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+<get_vts filter='modification_time>1706848198' details='1'/>
+``` 
+
+***With Scanner API***
+
+It is possible to get a list of supported OIDs, the whole feed information or information of a single VT. Filters are not implemented/supported.
+``` cmd
+curl --verbose --request GET http://127.0.0.1:3000/vts -H "X-API-KEY: changeme"
+curl --verbose --request GET http://127.0.0.1:3000/vts?information=true -H "X-API-KEY: changeme"
+curl --verbose --request GET http://127.0.0.1:3000/vts/1.3.6.1.4.1.25623.1.1.5.2014.322 -H "X-API-KEY: changeme"
```
[the remark this answers is not in the document]
Comment (jjnicola) [reply rust/openvasd/doc/osp-cmd-equivalence.md]: As talked, fix will be done within an independent PR. Documentation is kept on this document but it doesn't belong to the API specification.

File: rust/openvasd/doc/osp-cmd-equivalence.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+Results in Openvasd are stored with a result ID. This ID is useful to fetch results by ranges. Giving no ranges, all results are included in the response.
+
+``` cmd
+Method: GET
+Endpoint: /scans/{scan_id}/results
+
+Parameter scan_id: is de Scan ID
+``` 
+
```
Comment (Kraemii) [inline rust/openvasd/doc/osp-cmd-equivalence.md:184]:
```suggestion
```
I would remove the first request example

File: rust/openvasd/doc/osp-cmd-equivalence.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+    }
+  ]
+}
+
+```
+where "simple_scan_ssh_only.json" is the scan config. This HTTP request will get a response from the server which includes a scan_id. Later, every operation done on this task, must be done using the received scan_id. So for starting the stored scan you use the following POST method in the URL, passing the action as json object.
+
+``` cmd
+Method: POST
+Endpoint: /scans/{scan_id}
```
Comment (Kraemii) [inline rust/openvasd/doc/osp-cmd-equivalence.md:118]: Description of scan_id is missing

File: rust/openvasd/doc/osp-cmd-equivalence.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+
+***With OSP***
+``` xml
+<stop_scan scan_id='97079ee9-8917-49da-aa4f-4ef95f757ac1'/>
+``` 
+
+***With Scanner API***
+``` cmd
+Method: POST
+Endpoint: /scans/{scan_id}
```
Comment (Kraemii) [inline rust/openvasd/doc/osp-cmd-equivalence.md:137]: Description of scan_id is missing

**Review by Kraemii — CHANGES_REQUESTED** [review]

**Review by Kraemii — APPROVED** [review]

**Review by ArnoStiefvater — APPROVED** [review]

**Pull request merged**